consoles/video_stream.pm

```diff
@@ -225,6 +228,28 @@ sub send_key_event ($self, $key, $press_release_delay) {
       or die "failed to send '$key' input event";
 }
 
+sub type_string ($self, $args) {
+    my $keyboard_device_url = $bmwqemu::vars{GENERAL_HW_PICO_KEYBOARD_URL};
+    if ($keyboard_device_url) {
+        # send keyboard events using RPi Pico W based keyboard emulator
```

Comment:
This code doesn't seem very RPi Pico W specific at all. You're just making an HTTP request here. I suggest to document the interface of the HTTP server and also the variables you introduce. A good place to add such backend-specific documentation would be https://github.com/os-autoinst/os-autoinst/blob/master/doc/backends.md. Of course you can also give a bit more details about your specific setup there and e.g. provide a link to the code of the specific HTTP server you're using.